back/app/controllers/web_api/v1/ideas_controller.rb

```diff
@@ -389,12 +389,12 @@ def serialization_options_for(ideas)
         end
       user_followers ||= {}
       {
-        params: jsonapi_serializer_params(vbii: reactions.index_by(&:reactable_id), user_followers: user_followers, pcs: ParticipationPermissionsService.new),
+        params: jsonapi_serializer_params(vbii: reactions.index_by(&:reactable_id), user_followers: user_followers, pcs: IdeaPermissionsService.new),
         include: include
       }
     else
       {
-        params: jsonapi_serializer_params(pcs: ParticipationPermissionsService.new),
+        params: jsonapi_serializer_params(pcs: IdeaPermissionsService.new),
```

Comment:
Not clear if there is any benefit in passing the service in as a param, but have left it for now